dashboard/src/components/DeploymentForm/DeploymentForm.test.tsx

```diff
@@ -32,7 +31,8 @@ describe("renders an error", () => {
         selected={{ error: new NotFoundError() } as IChartState["selected"]}
       />,
     );
-    expect(wrapper.find(NotFoundErrorPage).exists()).toBe(true);
+    expect(wrapper.find(ErrorSelector).exists()).toBe(true);
+    expect(wrapper.find(ErrorSelector).html()).toContain("Chart &quot;foo&quot; (1.0.0) not found");
```

Comment:
I would avoid using mount in case there are a lot of components to dig into, but if there is only one like here, I think it's okay to use `.html()` here. Thanks!